manifests/gmetad.pp

```diff
@@ -1,17 +1,17 @@
 # @summary ganglia::gmetad
 #   Manages ganglia gmond & gmetad daemons + web front end
 # 
-# @param [Boolean] all_trusted defaults to: false
-# @param [Tuple] clusters defaults to: [ { 'name' => 'my cluster', 'address' => 'localhost' } ]
-# @param [String] gridname defaults to: `undef`
-# @param [Tuple] rras see README.md
-# @param [Array] trusted_hosts `Array of Strings` defaults to: '[]'
-# @param [String] gmetad_package_name
-# @param [String] gmetad_service_name
-# @param [String] gmetad_service_config
-# @param [String] gmetad_user
-# @param [Integer] gmetad_hostnames_case defaults to: 0
-# @param [String] gmetad_status_command
+# @param [Enum['on','off']] $all_trusted
```

Comment:
I don't think the types need to be in the rubydoc since they will be picked up from the code?

Reply:
You're probabley right, I was looking through it and it seems like yard had a few issues with generating a REFERENCE.md file because of the params lookup I think. Anyway, I'll remove this and stick to just naming the top level data type

Reply:
oh wait, what about the some of the gmond.pp params ie:
    # @param [Variant[String, Tuple]] $gmond_package_name

Reply:
Chatting with @natemccurdy on slack pointed out to just let the docs lookup the type itself. So removing all types seems a better strategy. Fixing that up as well.

Reply:
Yep -- that's what I was suggesting.